lib/Kconfig

```diff
@@ -22,4 +22,6 @@ rsource "modem_info/Kconfig"
 
 rsource "pdn_management/Kconfig"
 
+rsource "zzhc/Kconfig"
+
```

Comment:
The two commits should be one.

It doesn't make sense to include one without the other.

Reply:
Will squash them into single commit.

Reply:
Squashed.